packages/editor/src/functions/EditorControlFunctions.ts

```diff
@@ -83,6 +84,13 @@ import { SelectionState } from '../services/SelectionServices'
 const tempMatrix4 = new Matrix4()
 const tempVector = new Vector3()
 
+const isInActiveScene = (entity: Entity) => {
```

Comment:
can we put this on EditorState as a helper funciton? Would be nice to reuse in a few places i reckon